.circleci/config.yml

```diff
@@ -25,19 +25,27 @@ jobs:
       name: docker-executor
       python_version: "<<parameters.python_version>>"
     parallelism: 4
+    working_directory: ~/project/api
     steps:
-      - checkout
+      - checkout:
+          path: ~/project
       - run:
           name: Install Dependencies
-          command: cd api && poetry install --no-root && rm -rf $POETRY_CACHE_DIR
+          command: poetry install --no-root
+      - run:
+          name: Set PYTHONPATH
+          command: echo 'export PYTHONPATH=$PYTHONPATH:.' >> $BASH_ENV
       - run:
           name: Run tests
-          command: cd api && poetry run pytest ../tests
+          command: |
+            poetry run pytest ../tests || poetry run pytest --last-failed ../tests
```

Comment:
I think we can make them pyinvoke tasks so that we can easily test them in the future. Non-blocking. Will do it after this PR

Reply:
Sure. Would you be creating an issue for this?

Reply:
https://github.com/astronomer/ask-astro/issues/89